### PR TITLE
Update about screen content

### PR DESCRIPTION
**Pull request by TinasheMzondiwa**
# What did you change:

Updated about screen content

# Screenshot/GIFs of this change

Screenshot_20210513-214657

# Merge checklist

- [ ] Automated (unit/ui) tests
- [x] Manual tests

### CODE REVIEW

File: app/src/main/java/com/cryart/sabbathschool/ui/about/AboutActivity.kt [hunk 1]
```diff
@@ -48,11 +51,35 @@ class AboutActivity : SSColorSchemeActivity() {
...
+        return when (item.itemId) {
+            android.R.id.home -> {
+                finishAfterTransition()
+                true
+            }
+            R.id.ss_action_share -> {
+                val shareIntent = ShareCompat.IntentBuilder(this)
+                    .setType("text/plain")
+                    .setText(
+                        String.format(
```
Comment (merlin-zaraza) [inline app/src/main/java/com/cryart/sabbathschool/ui/about/AboutActivity.kt]: Use Kotlin string template instead of format

**Review by merlin-zaraza — APPROVED** [review]
I like this new design

**Review by VitalikL — CHANGES_REQUESTED** [review]
@TinasheMzondiwa The typeface for the SSPM text is different from Adventech typeface. Please change the SSPM text to exactly match Adventech copy. 

- Font typeface
- Font size
- Letter spacing
- Color

**Comment by VitalikL** [comment]
@TinasheMzondiwa Another issue is that the topbar / navbar is different from other views, ex Settings. In Settings, the navbar adopts the primary color, but About us has permanent white style. Please make it similar to Settings to be consistent

File: app/src/main/java/com/cryart/sabbathschool/ui/about/AboutActivity.kt (hunk 1 above)
Comment (TinasheMzondiwa) [reply app/src/main/java/com/cryart/sabbathschool/ui/about/AboutActivity.kt]: Done

**Review by VitalikL — APPROVED** [review]
🚀

**Pull request merged**